Documentation/observability/hubble/configuration/tls.rst

```diff
@@ -343,3 +343,39 @@ Hubble metrics API.
   provide the path to the CA certificate. When using mTLS you will also need to
   provide a client certificate and key signed by the CA certificate for
   Prometheus to authenticate to the Hubble metrics API.
+
+.. _hubble_api_tls:
+
+Access the Hubble API with TLS Enabled
+======================================
+
+The examples adapted from :ref:`hubble_cli`.
+
+Before you can access the Hubble API with TLS enabled, you need to obtain the
+CA certificate from the secret that was created when enabling TLS. The
+following examples demonstrate how to obtain the CA certificate and use it to
+access the Hubble API.
+
+Run the command below to obtain the CA certificate from the ``hubble-relay-server-certs`` secret:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system get secret hubble-relay-server-certs -o jsonpath='{.data.ca\.crt}' | base64 -d > hubble-ca.crt
+
+After obtaining the CA certificate you can use the  ``--tls`` to enable TLS and ``--tls-ca-cert-files`` flag to specify the CA certificate. Additionally, when port-forwarding to Hubble Relay, you will need to specify the ``--tls-server-name`` flag.
```

Comment:
Do you mind wrapping this line and adding a colon at the end, since it leads into an example command?

```suggestion
After obtaining the CA certificate you can use the  ``--tls`` to enable TLS and ``--tls-ca-cert-files`` flag to specify the CA certificate. Additionally, when port-forwarding to Hubble Relay, you will need to specify the ``--tls-server-name`` flag:
```